class AlreadyExists(Exception): pass
class NoNameProvided(Exception): pass
